.env.example

```diff
@@ -0,0 +1,2 @@
+CF_DNS_API_TOKEN=<cloudflare api token, Zone/DNS/Edit permissions>
```

Comment:
There’s another CF variable for finer tuning of Cloudflare permissions, I used it before... will look it up